### PR TITLE
English review

### PR DESCRIPTION
**Pull request by Nuscly**
english review

### CODE REVIEW

File: content/md/cloud/1-set-up-flexibility.md [hunk 1]
```diff
@@ -10,47 +10,47 @@ related: access-akeneo-flexibility, get-akeneo-pim-enterprise-archive
...
+2. Go to the Cloud instance tab, and open the instance with the status `Waiting for setup`, by clicking on the instance line.
 3. Then fill the form with the missing prerequisites in order to create the Cloud server. 
-4. Choose a `Domain prefix`, which is going to be at the beginning of your URL instance (ex: https://yourdomainprefix.cloud.akeneo.com). An instant check is done to see if the name is available.
-5. Select an `Installation locale`, your cloud instance language
-6. Select an `Installation location` for your cloud instance hosting location that is closest to you to reduce latency access.
-7.  Select a `Target number of products` for your catalog.
-8. Once the information is inputted, click `Request an Instance`
+4. Choose a `Domain prefix`, which is going to appear at the beginning of your URL instance (ex: https://yourdomainprefix.cloud.akeneo.com). An instant check runs to see if the name is available.
+5. Select an `Installation locale`, which is your cloud instance language
+6. Select an `Installation location` for your cloud instance hosting location. It should the closest to you to reduce latency access.
```
Comment (Nuscly) [inline content/md/cloud/1-set-up-flexibility.md]:
```suggestion
6. Select an `Installation location` for your cloud instance hosting location. It should be the closest to you to reduce latency access.
```

Comment (vilain-corentin) [reply content/md/cloud/1-set-up-flexibility.md]: Should used for sense of advice
Should be used when you have to need something
Depends on how we want to put it

File: content/md/cloud/set-up-serenity.md [hunk 2]
```diff
@@ -25,27 +25,27 @@ The setup should be done for each of your **production** or **sandbox** instance
 
 1. Choose a `Domain prefix`, which is going to be your URL Instance (ex: https://yourdomain.cloud.akeneo.com). An instant check is done to see if the name is available.
 1. Select an `Installation locale`, your cloud instance language
-1. Select an `Installation location`, for your cloud instance hosting location that is closest to you to reduce latency access.
+1. Select an `Installation location`, for your cloud instance hosting location, it should be the closest to you to reduce latency access.
```
Comment (vilain-corentin) [inline content/md/cloud/set-up-serenity.md]: it should the closest...
if we want to stay "iso"

File: content/md/onboarder/1-set-up-onboarder.md [hunk 3]
```diff
@@ -40,35 +40,35 @@ The [technical prerequisites](https://docs.akeneo.com/latest/onboarder/prerequis
 2. Choose a `Domain prefix`, which is going to be at the beginning of your URL instance (e.g: https://retailer-yourdomainprefix.cloud.akeneo.com). An instant check is done to see if the name is available.
 3. Select an `Installation locale` for your Onboarder.
 4. Select an `Installation location` for your Onboarder instance. Choose the closest location from you to reduce latency access.
```
Comment (vilain-corentin) [inline content/md/onboarder/1-set-up-onboarder.md]: It should the closest location from you to reduce latency access.

**Pull request merged**